.tekton/pipeline-service-ckcp-testing.yaml

```diff
@@ -0,0 +1,295 @@
+            - name: ckcp-setup
+              image: quay.io/xinjiang/kubectl:1.25
+              resources:
+                requests:
+                  memory: 500Mi
+                  cpu: 300m
+              script: |
+                #!/usr/bin/env bash
+                ## Produce a ckcp_setup.sh to assist execte ckcp sciprt
+                cat <<'EOF' > $(workspaces.source.path)/ckcp_setup.sh
+                #!/usr/bin/env bash
+                set -o errexit
+                set -o pipefail
+                export PATH=/usr/local/go/bin:$PATH
+                export PATH=$(go env GOPATH)/bin:$PATH
+                export ALLOW_ROOTLESS=true
+
+                echo "Start installing kcp plugin ..."
+                CKCP_DIR=$(find "$PWD" -type f -name openshift_dev_setup.sh -exec dirname {} +)
+                KCP_TAG="$(yq '.images[] | select(.name == "kcp") | .newTag' "$CKCP_DIR/openshift/overlays/dev/kustomization.yaml")"
+                echo "kcp tag:" $KCP_TAG
+                KCP_TAG_SHORT="$(printf '%s' "$KCP_TAG" | sed 's/v//')"
+                curl -OL https://github.com/kcp-dev/kcp/releases/download/${KCP_TAG}/kubectl-kcp-plugin_${KCP_TAG_SHORT}_linux_amd64.tar.gz && tar -C /usr/local -xzf kubectl-kcp-plugin_${KCP_TAG_SHORT}_linux_amd64.tar.gz
+                kubectl-kcp --version
+
+                echo "\n\nStart ckcp setup ..."
+                cd $CKCP_DIR && ./openshift_dev_setup.sh -d
+                EOF
+
+                chmod +x "$(workspaces.source.path)/ckcp_setup.sh"
+
+                ## start executing ckcp setup script
+                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+
+                echo "Copy pipeline-service source codes to podman container"
+                kubectl cp $(workspaces.source.path) default/podman:/source
+
+                echo "Copy kubeconfig file to podman container"
+                kubectl cp "$(workspaces.kubeconfig-dir.path)/kubeconfig" default/podman:/kubeconfig
+
+                echo "Start to set up ckcp"
+                kubectl -n default exec pod/podman -- sh -c "export KUBECONFIG=/kubeconfig && /source/ckcp_setup.sh"
+    finally:
+      - name: destroy-cluster
+        params:
+          - name: deploy-cluster-status
+            value: "$(tasks.deploy-cluster.status)"
+          - name: namespace
+            value: "$(params.namespace)"
+          - name: cluster-name
+            value: "$(tasks.produce-cluster-name.results.cluster-name)"
+        workspaces:
+          - name: kubeconfig-dir
+            workspace: kubeconfig-dir
+        taskSpec:
+          params:
+            - name: deploy-cluster-status
+            - name: namespace
+            - name: cluster-name
+          workspaces:
+            - name: kubeconfig-dir
+          kind: ClusterTask
+          steps:
+            - name: destroy
+              image: quay.io/xinjiang/kubectl:1.25
```

Comment:
Change this to the official image. If this is a custom image you've built, then please upload this to quay.io/redhat-pipeline-service organization.

Reply:
ah, thanks for pointing this issue. I forgot it

Reply:
fixed, same as above